apps/console/src/features/identity-providers/components/wizards/authenticator-create-wizard-factory.tsx

```diff
@@ -74,6 +83,16 @@ export const AuthenticatorCreateWizardFactory: FunctionComponent<AuthenticatorCr
         open,
         onWizardClose,
         type,
+        /**
+         * Added this because {@link IdentityProviderTemplateSelectPage} currently
+         * unable to handle grouped templates properly. {@link selectedTemplate}
+         * will be a grouped identity provider with sub templates. Even though, the
+         * grouping logic implemented and is in place {@link getTemplate} method
+         * keeps failing to set the correct grouped template to this state.
+         *
+         * FIXME: As a part of https://github.com/wso2-enterprise/asgardeo-product/issues/3878
+         */
+        selectedTemplate: parentSel,
```

Comment:
Let's add a meaningful alias here.
```suggestion
        selectedTemplate: parentSel,
```

Reply:
+1 Addressed this in commit 63790c8